src/topology/sheaves/locally_surjective.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2022 Sam van Gool and Jake Levinson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sam van Gool, Jake Levinson
+-/
+
+import topology.sheaves.presheaf
+import topology.sheaves.stalks
+
+/-!
+
+Locally surjective maps of presheaves.
+
+Let X be a topological space, ℱ and 𝒢 presheaves on X, T : ℱ ⟶ 𝒢 a map.
```

Comment:
```suggestion
Let `X` be a topological space, `ℱ` and `𝒢` presheaves on `X`, `T : ℱ ⟶ 𝒢` a map.
```

I think it would look better on the documentations page if variables are surrounded in backticks.
Same for the ones below.